pikachu_education/lib/utils/management_image.dart

```diff
@@ -3,4 +3,6 @@ class ManagementImage {
   static String defaultAvatar = 'assets/image/pikachu.png';
   static String defaultAvatar2='assets/image/pikachu_itachi.png';
   static String iconTag='assets/image/icon_tag.png';
+  static String logoGoogle = 'assets/image/google_logo.png';
+  static String logoFacebook = 'assets/image/facebook_logo.png';
```

Comment:
Please use the DRY (Don't Repeat Yourself) mindset.
You can see the duplicated `assets/image/`. Please create  common String for that